### PR TITLE
2863 review contributing guide

### PR DESCRIPTION
**Pull request by timothy-nunn**
I have done a first iteration of how our contributing guide could look. I welcome recommendations/ changes on the new guide. This is heavily influenced by [bluemira's contributing guide](https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/develop/CONTRIBUTING.md).

We will want to consider how changes to the CI (https://github.com/ukaea/PROCESS/issues/2867) will affect this workflow.

### CODE REVIEW

**Review by jonmaddock — CHANGES_REQUESTED** [review]
This is great Tim, thanks for sorting this out; it reads much better and is up-to-date now.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,54 +1,38 @@
...
+
+## Creating an issue
+Issues can be used to report bugs or request features and improvements. We ask you help us manage our issues by:
+* Verifying your issue is not a duplicate of another issue; in this case, we welcome your contribution as a reply to the issue.
+* Ensure you completely describe the bug/feature/problem and complete the given templates with appropriate detail.
+
+## Submitting a pull request
+Please discuss any feature ideas you have with the developers before submitting them, as you may not be aware of parallel development work taking place, or implementation decisions / subtleties which are relevant. The ideal workflow for submitting a pull request is as follows:
+
+* Discuss the feature with a core PROCESS developer.
```
[the remark this answers is not in the document]
Comment (timothy-nunn) [reply CONTRIBUTING.md:20]: There are two ways, we could mention both or choose on:
* Use the emails listed on the README (you, James, Stuart, Michael).
* Create an issue that is a feature request.

Do you have a preference?

[the remark this answers is not in the document]
Comment (timothy-nunn) [reply CONTRIBUTING.md:20]: In fact, I will just link this bullet point back to the top that (now) says 'issue, discussion, email'

**Review by jonmaddock — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonmaddock) [reply CONTRIBUTING.md]: Great!

**Pull request merged**